hyperspy/signal.py

```diff
@@ -374,7 +374,7 @@ def _plot_factors_or_pchars(self, factors, comp_ids=None,
         else:
             return f
 
-    def _plot_loadings(self, loadings, comp_ids=None, calibrate=True,
```

Comment:
This was not necessary to change the definition of this function, but it does not have any consequences. It just makes the code a bit less let consistence with the definition of `_plot_factors_or_pchars`.